docs/index.asciidoc

```diff
@@ -337,6 +338,18 @@ If encountered as response codes this plugin will retry these requests
 
 Timeout (in seconds) to wait for data on the socket. Default is `10s`
 
+[id="plugins-{type}s-{plugin}-ssl_verification_mode"]
+===== `ssl_verification_mode`
+
+  * Value type is <<string,string>>
+  * Supported values are: `full`, `none`
+  * Default value is `full`
```

Comment:
I am thinking what does `full` mean. Beats has a nice [doc](https://www.elastic.co/guide/en/beats/filebeat/current/configuration-ssl.html)
Maybe a good reference.

Reply:
OK, thought I would keep it short but we could copy the description from Beats' docs ...